app/assets/javascripts/crops/svg_button.js

```
@@ -0,0 +1,25 @@
+var MISSING_PROP =
+  "You forgot to attach an `on-change=''` function to the <svg-button/> directive.";
```

Yeah, I'm always thrown off by silent errors with missing props.